book/src/guides/injection.md

```diff
@@ -0,0 +1,56 @@
+# Adding Injection Queries
+
+Writing language injection queries allows one to highlight a specific node as a different language.
+In addition to the [standard](upstream-docs) language injection options used by tree-sitter, there
+are a few Helix specific extensions that allow for more control.
+
+And example of a simple query that would highligh all strings as bash:
+```scm
+((string_expression (string_fragment) @injection.content)
+  (#set! injection.language "bash"))
+```
```

Comment:
It's probably worth noting here that the language is Nix for this query. Other languages won't have the same node names so this would make an invalid query